src/v/bytes/iobuf.h

```diff
@@ -139,8 +139,11 @@ class iobuf {
      */
     void append_fragments(iobuf);
 
-    /// \brief trims the back, and appends direct.
-    void append_take_ownership(std::unique_ptr<fragment>);
+    /**
+     * Append a fragment. The fragment will be trimmed.
```

Comment:
If there is a last fragment, it will be trimmed.

Reply:
I think the note about the fragment being trimmed doesn't really even need to be documented. It's an internal detail

Reply:
> I think the note about the fragment being trimmed doesn't really even need to be documented. It's an internal detail

Agree

Reply:
fixed comment.